packages/reports/addon/components/common-actions/schedule.hbs

```diff
@@ -3,15 +3,16 @@
 <DenaliModal
   @isOpen={{this.showModal}}
   @onClose={{queue this.closeModal (fn @onRevert this.localDeliveryRule)}}
+  class="schedule__modal"
   ...attributes as |Modal|
 >
   <Modal.Content>
-    <h3 class="schedule__modal-header m-b-10">Schedule {{this.localDeliveryRule.deliveryType}}</h3>
+    <h3 class="schedule__modal__header m-b-10">Schedule {{this.localDeliveryRule.deliveryType}}</h3>
```

Comment:
```suggestion
    <h3 class="schedule__modal-header m-b-10">Schedule {{this.localDeliveryRule.deliveryType}}</h3>
```

Reply:
All the classes in the model should be `scheduke__model-{element}`

Reply:
gotcha, I'll switch em back.